frontend/src/components/story/StoryPeriod.vue

```diff
@@ -1,14 +1,16 @@
 <template>
-  <v-expansion-panels v-model="expandedDays" accordion flat multiple>
+  <v-expansion-panels ref="dayPanels" v-model="expandedDays" accordion flat multiple>
     <story-day
-      v-for="day in period.days().items"
+      v-for="day in sortedDays"
       :key="day._meta.self"
       :day="day"
       :editing="editing"
     />
   </v-expansion-panels>
 </template>
 <script>
+import { sortBy } from 'lodash'
+import { nextTick } from 'vue'
```

Comment:
You can just use `this.$nextTick` anywhere in a Vue component, no need to import it